src/pages/index.js

```diff
@@ -33,60 +47,165 @@ const cardSection = new Section({
   }
 }, ".cards");
 const popupWithImage = new PopupWithImage('.modal_type_image');
-const api = new Api({
-  baseUrl: "https://around-api.en.tripleten-services.com/v1",
-  headers: {
-    authorization: "32400e4e-bdb6-4554-9f2b-339b0c55f4e6",
-    "Content-Type": "application/json"
-  }
-});
 
 
-// define functions
-function renderProfileDetails(){
-  const { name, description } = userInfo.getUserInfo();
-  nameInput.value = name;
-  jobInput.value = description;
+/* ------------------------------------------------------ */
+/*                    define functions                    */
+/* ------------------------------------------------------ */
+
+function renderProfileDetails(modal){
+  const { name, about } = userInfo.getUserInfo();
+  modal.setInputValues({name, about})
 };
 
 function handleCardAdd(data){
-  const { title, url } = data
-  const cardElement = createCard({title, url})
-  cardSection.addItem(cardElement)
-  cardModal.close()
+  this.renderLoading(true)
```

Comment:
`this` should **not** be used outside classes in functions in `index.js` . This context is easy to lose. You need to find other ways of passing data.

Here you need to use `cardModal` instead of `this`

`cardModal.renderLoading(true)`

**Please, correct it everywhere**